src/backends/torch/torchmodule.h

```diff
@@ -195,7 +195,10 @@ namespace dd
     torch::nn::Linear _linear = nullptr;
 
     torch::Device _device;
-    int _linear_in = 0; /**<id of the input of the final linear layer */
+    bool _training = false; /**<true if model is in training mode */
+    int _linear_in = 0;     /**<id of the input of the final linear layer */
```

Comment:
does this only apply to the object detection models from torchvision ?

Reply:
`_training` field is set when we call the method train(). Currently it's used only for models that have different output on training and eval mode, ie detection models.